code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -769,16 +769,16 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	else
 		L = get(src, /mob/living/silicon)
 
-	if(L)
+	if(L && L.stat != UNCONSCIOUS)

```

Comment:
Yeah if I'm dead and waiting in my body to be cloned, there is no harm in me seeing PDA messages